static/styles/banners.css

```diff
@@ -0,0 +1,65 @@
+body{
+    margin:0px;
+    /* font-family: Dosis; */
+}
+@media (min-width: 1025px) {
+  body{
+    max-width: 80%;
+    padding: 0 15%;
+    /* display: flex;
+    flex-direction: column;
+    justify-content: center */
+  } 
+  }
+ol,ul{
+    font-family: Dosis;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    text-align: center;
+}
+.carousel_viewport{
+    /* font-family: Dosis; */
+    display: flex;
+    overflow-x: scroll;
+    scroll-behavior: smooth;
+}
+.categories>ul>li{
+    height:200px;
+    /* display: inline-block; */
+    width: 100%;
+    margin: 0px 0px;
+    padding:20px;
+    box-shadow:  0 6px 15px -4px rgba(0,0,0,0.7);
+    display: flex;
+    justify-content: space-around;
+}
+.categories>ul>li:nth-child(even){
+    flex-direction: row-reverse;
+}
+.categories>ul>li>img{
+    width:30%;
+}
+/* .categories>ul>li:nth-child(odd)>img{
+    float: left;
+}
+.categories>ul>li:nth-child(odd)>aside{
+    display: inline;
+    float: unset;
+}
+.categories>ul>li:nth-child(even)>img{
+    float:right;
+}
+.categories>ul>li:nth-child(even)>aside{
+    /* display: inline;
+    float: unset;
+} */
+
+.categories>ul>li>section>button{
+    background-color: #e91e63;
```

Comment:
would appreciate if you could take care of the indentation. Add a space after/before every >. Add space before {